web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -615,17 +678,41 @@
       return this.getAttribute('time-step') || 30;
     }
 
-    /**
-     * list of all CelementsDateTimeField siblings of this (including this) within the defined interdependence-wrapper
-     */
-    get siblings() {
+    get #interdependenceWrapper() {
       try {
-        const wrapper = this.closest(this.getAttribute('interdependence-wrapper'));
-        return [...wrapper?.querySelectorAll('cel-input-date-time, cel-input-date, cel-input-time') || []];
+        return this.closest(this.getAttribute('interdependence-wrapper') || 'form');
       } catch (exp) {
-        console.debug('siblings: no valid parent defined');
-        return [];
+        console.debug('interdependenceWrapper: no valid parent defined');
+      }
+    }
+
+    get #interdependenceRoles() {
+      return this.getAttribute('interdependence-role')?.split(',') || [];
+    }
+
+    update(data) {
+      console.debug('update', this, data);
+      data = data || {};
+      for (const key in data) {

```

Comment:
this looks dangerous to me, because you can change any internal field with a prepared event data. Can you please filter the keys to an allowed-key list?